src/events/messageUpdate.ts

```diff
@@ -1,22 +1,30 @@
 import FluorineClient from '@classes/Client';
 import Embed from '@classes/Embed';
 import { Message } from 'discord.js';
-import r from 'rethinkdb';
-import { SettingsType } from 'types/settings';
+import { Config } from 'types/databaseTables';
+
 export async function run(client: FluorineClient, oldMessage: Message, newMessage: Message) {
     if (!newMessage || newMessage.content === oldMessage.content) {
         return;
     }
-    // @ts-ignore
-    const settings: SettingsType = await r.table('config').get(newMessage.guild?.id).run(client.conn);
-    if (!settings.logs || !settings.logsChannel) {
+
+    const [settings] = (
+        await client.db.query<Config>('SELECT logs_enabled, logs_channel FROM config WHERE guild_id = $1', [
+            BigInt(oldMessage.guild.id)
+        ])
+    ).rows;
+
+    if (!settings.logs_enabled || !settings.logs_channel) {
         return;
     }
-    const channel = client.channels.cache.get(settings.logsChannel);
+
+    const channel = client.channels.cache.get(settings.logs_channel.toString());
     if (!channel.isText()) {
```

Comment:
```suggestion
    if (!channel?.isText()) {
```
otherwise will fail if the channel doesnt exist